src/components/Table/GridTable.tsx

```diff
@@ -1068,7 +1077,8 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
   } = props;
 
   const { rowState } = useContext(RowStateContext);
-  const isActive = useComputed(() => rowState.activeRowId === `${row.kind}_${row.id}`, [row, rowState]);
+  const rowId = `${row.kind}_${row.id}`;
+  const isActive = useComputed(() => rowState.activeRowId === rowId, [row, rowState]);
```

Comment:
**nit:** `rowId` should now be part of the dependency array.

Reply:
question, does'nt  putting rowId on that dependency array will make it fire useComputed every render? since is a variable that get re-defined on every render, also, if we already have row object as a dependency, then if the row changes we will get the new rowId, and rowId can not change unless row changed

Reply:
Since `rowId` is a string, it will be properly evaluated as `"rowId" === "rowId"`. There should be no difference, and would not cause the `useComputed` to fire again.